packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -20,17 +20,12 @@ class CreateCommand extends Command {
   final String name = 'create';
 
   @override
-  final String description = 'Create a new Flutter project.';
+  final String description = 'Create a new Flutter project.\nIf run on a project that already exists, this will repair the project, recreating any files that are missing.';
```

Comment:
```
$ flutter create --help
Building flutter tool...
Create a new Flutter project.
If run on a project that already exists, this will repair the project, recreating any files that are missing.

Usage: flutter create <output directory>
-h, --help                     Print this usage information.
    --[no-]pub                 Whether to run "pub get" after the project has been created.
                               (defaults to on)

    --[no-]with-driver-test    Also add a flutter_driver dependency and generate a sample 'flutter drive' test.

Run "flutter help" to see global options.
$ flutter --help
Manage your Flutter app development.

Usage: flutter <command> [arguments]

Global options:
-h, --help            Print this usage information.
-v, --verbose         Noisy logging, including all shell commands executed.
-d, --device-id       Target device id.
    --version         Reports the version of this tool.
    --flutter-root    The root directory of the Flutter repository. Uses $FLUTTER_ROOT if set,
                      otherwise defaults to a value derived from the location of this tool.
                      (defaults to "...")

Available commands:
  analyze   Analyze the project's Dart code.
  build     Flutter build commands.
  create    Create a new Flutter project.
  devices   List all connected devices.
  doctor    Show information about the installed tooling.
  drive     Runs Flutter Driver tests for the current project.
  help      Display help information for flutter.
  install   Install a Flutter app on an attached device.
  listen    Listen for changes to files and reload the running app.
  logs      Show log output for running Flutter apps.
  refresh   Build and deploy the Dart code in a Flutter app (Android only).
  run       Run your Flutter app on an attached device.
  stop      Stop your Flutter app on an attached device.
  test      Run Flutter unit tests for the current project (Linux only).
  trace     Start and stop tracing for a running Flutter app (Android only).
  upgrade   Upgrade your copy of Flutter.

Run "flutter help <command>" for more information about a command.
Run "flutter -h -v" for verbose help output, including less commonly used options.

[✓] The Android toolchain is fully installed.
[ ] The Atom development environment is not installed.

Run "flutter doctor" for information about installing additional components.
$
```